x-pack/plugins/fleet/common/constants/output.ts

```diff
@@ -58,14 +74,40 @@ export const kafkaAcknowledgeReliabilityLevel = {
   DoNotWait: 'Do not wait',
 } as const;
 
-export const DEFAULT_OUTPUT_ID = 'fleet-default-output';
-
-export const DEFAULT_OUTPUT: NewOutput = {
-  name: 'default',
-  is_default: true,
-  is_default_monitoring: true,
-  type: outputType.Elasticsearch,
-  hosts: [''],
-};
-
-export const LICENCE_FOR_PER_POLICY_OUTPUT = 'platinum';
+export const kafkaSupportedVersions = [
```

Comment:
I'm wondering if the versions could be queried from somewhere dynamically?

Reply:
I gave it some thought, but I've decided against it. We would need to expose our own API for this and still hardcode the values because we might not want to allow a version that was just released and not yet tested by us.